material/src/main/java/org/odk/collect/material/ErrorsPill.kt

```diff
@@ -0,0 +1,35 @@
+package org.odk.collect.material
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import com.google.android.material.color.MaterialColors
+import org.odk.collect.androidshared.system.ContextUtils
+
+class ErrorsPill(context: Context, attrs: AttributeSet?) : MaterialPill(context, attrs) {
```

Comment:
This is really nice! I think it'd be best for this to live in `androidshared`, as it's not really "missing" from the Material Components library and is more a Collect specific piece of UI.

Also, while we're here I think it'd be good to pull out an `ErrorsPillTest` so that the existing (`SelectionSummarySheetTest` and `IsntanceListItemViewTest`) and new tests can just assert on the state of `ErrorPill` instead of needing to know anything about how it works.

Reply:
> This is really nice! I think it'd be best for this to live in androidshared, as it's not really "missing" from the Material Components library and is more a Collect specific piece of UI.

the `material` module already uses `androidshared`. If I move `ErrorsPill` to `androidshared` then I would need to add the `material` module as a dependency (to extend `MaterialPill`) and that will be a circular dependency.

Reply:
Done.